main.go

```diff
@@ -29,6 +29,11 @@ func main() {
 			Usage:  "passphrase for the key",
 			EnvVar: "PLUGIN_PASSPHRASE,GPGSIGN_PASSPHRASE,GPG_PASSPHRASE",
 		},
+		cli.BoolFlag{
```

Comment:
As this is a breaking change to the current behavior I would prefer a `Value: true,` attribute for the flag to enable it by default.

Reply:
There is no ``Value`` attribute for ``BoolFlag``. I'm going to change it for a "non-armor" Bool flag instead.

Reply:
Ah, it's named `Default`, not `Value`... Sorry!

Reply:
Ok, we are still using urfave/cli v1 here, so it works totally different.

```
-		cli.BoolFlag{
+		cli.BoolTFlag{
```

Reply:
Thanks, that's what I was looking for